core/commonTest/src/kotlinx/serialization/UmbrellaTypes.kt

```diff
@@ -69,8 +69,8 @@ data class ArraysUmbrella(
 
 @SharedImmutable
 val umbrellaInstance = TypesUmbrella(
-    Unit, true, 10, 20, 30, 40, 50.1f, 60.1, 'A', "Str0", Attitude.POSITIVE, IntData(70),
```

Comment:
These fixes here just because wasm double parser slightly differs from other's targets.

Reply:
Yes, we hope to fix it later.